LiteFarmOrg/LiteFarm
# Lf 2695 reconfigure nodemon for api package to support shared code

This PR resolves https://lite-farm.atlassian.net/browse/LF-2695

There are two parts to this PR:
1. Is the developer tooling working?
2. Is the refactor of the sensor validation working?

To test part 1 of this PR:
1. Run both the webapp and the api
2. Make a change in the /shared package. The api should reload automatically, and if you are on a page which uses code from /shared in the webapp, that page should reload automatically as well (easiest way to verify this is to open the upload sensors modal and then modify something in the `parseSensorCsv` function in `shared/validation/sensorCSV.js`)

To test part 2 of this PR:
1. Try to upload an invalid csv file in english - you should get reasonable errors
2. Try again in another language - you should get reasonable errors
3. Try to upload a valid csv file in english - this should work (if the sensors are unclaimed)
4. Try to upload a valid csv file in another language - this should work (if the sensors are unclaimed)
